dev-libs/newt/files/newt-0.52.13-gold.patch

```diff
@@ -1,5 +1,5 @@
---- configure.ac~	2011-11-09 23:50:21.000000000 +0900
-+++ configure.ac	2011-11-09 23:54:54.139482817 +0900
+--- a/configure.ac	2011-11-09 23:50:21.000000000 +0900
++++ b/configure.ac	2011-11-09 23:54:54.139482817 +0900
```

Comment:
not important now, but if you're git/p1-izing the patches, could you also remove the dates? unnecessary bytes, add no value, and are just noisy